meta-luneos/recipes-navigation/geoclue/geoclue_%.bbappend

```diff
@@ -1,2 +1,5 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
 SRC_URI += "file://0001-Use-google-as-default-and-add-api-key.patch"
+
+# Avoid putting a dependency on modemmanager
+PACKAGECONFIG_remove = "3g modem-gps cdma"
```

Comment:
Remove operator makes it impossible to add later (e.g. from local.conf)

Adding just 
PACKAGECONFIG = "nmea lib"
would be more flexible.